src/main/java/com/nhnacademy/hexashoppingmallservice/entity/cart/Cart.java

```diff
@@ -0,0 +1,42 @@
+package com.nhnacademy.hexashoppingmallservice.entity.cart;
+
+import com.nhnacademy.hexashoppingmallservice.entity.Member;
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.awt.print.Book;
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Getter
+public class Cart {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int cartId;
+
+    @Setter
+    @Column(nullable = false)
+    private int cartAmount;
+
+    @ManyToOne
+    @JoinColumn(name = "member_id")
+    private Member member;
+
+    @ManyToOne
+    @JoinColumn(name = "book_id")
+    private Book book;
+
+
+//    @Builder
```

Comment:
builder 타입으로 통일. static of 매서드 이용
```suggestion
//    @Builder
```